### PR TITLE
Added glenn2vcf tool.

### PR DESCRIPTION
**Pull request by cmarkello**

### CODE REVIEW

File: glenn2vcf/test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python2.7
+# Charles Markello
+import subprocess
+import tempfile
+import unittest
+
+
+class TestVG(unittest.TestCase):
```
Comment (hannes-ucsc) [inline glenn2vcf/test.py]: You probably want to rename this.

File: glenn2vcf/build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:15.10
```
Comment (hannes-ucsc) [inline glenn2vcf/build/Dockerfile]: Is it a good idea to build on 15.10 and deploy to 14.04?

File: glenn2vcf/runtime/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:14.04
+
+MAINTAINER Charles Markello, cmarkell@ucsc.edu
+
+RUN mkdir /opt/glenn2vcf/
+WORKDIR /home/glenn2vcf
```
Comment (hannes-ucsc) [inline glenn2vcf/runtime/Dockerfile]: What's this for?

File: glenn2vcf/build/Dockerfile (hunk 2 above)
Comment (cmarkello) [inline glenn2vcf/build/Dockerfile]: Probably not, though I did test it and it works properly. I'll change the runtime environment to 15.10.

The main reason for it is an issue with installation of the correct version of gcc (gcc-4.9) that is only accessible to Ubuntu version 15.10 and later for some reason. It's only needed for compilation and the binary runs and works on 14.04, so the deployment shouldn't be affected by that.

For consistency, it makes sense to have them both be the same version. I did the same thing for the vg tool and will also change it.

Comment (jvivian) [inline glenn2vcf/build/Dockerfile]: An LTS version would probably be preferable, whether thats 14.04 or 16.04

Comment (hannes-ucsc) [inline glenn2vcf/build/Dockerfile]: For both, runtime and build-time containers, yes.

**Comment by hannes-ucsc** [comment]
Could you please squash and force push? We try very hard to maintain a clean history. Otherwise +1. And don't forget to remove the `needs work` label when you think this branch can be merged.

**Comment by cmarkello** [comment]
I don't believe I can remove the needs work label. I think I need permissions to do that.

**Comment by hannes-ucsc** [comment]
First line in commit message should be < 72 characters.

**Comment by hannes-ucsc** [comment]
<= 72

**Comment by hannes-ucsc** [comment]
Also, try again with the label. I fixed your permissions.

**Comment by hannes-ucsc** [comment]
http://chris.beams.io/posts/git-commit/

with the exception that we allow 72 characters in the first line. Not just 50 or 60.

**Pull request closed without merging**